### ISSUE TITLE
Support for AWS OTel Collector in Quarkus OpenTelemetry

### ISSUE THREAD

**Issue by syr** [quarkusio/quarkus-quickstarts#1116]
Hi, I tried to set up tracing using quarkus-opentelemetry-exporter-otlp by unchanged code of https://github.com/quarkusio/quarkus-quickstarts/tree/main/opentelemetry-quickstart 
while having AWS OTel Collector version: v0.17.0 running.

On rest resource access, traces seem to arrive in AWS OTel Collector, however I get an error

```
2022-05-19T13:40:17.841Z        debug   awsxrayexporter@v0.45.1/awsxray.go:54   TracesExporter  {"kind": "exporter", "name": "awsxray", "type": "awsxray", "name": "awsxray", "#spans": 1}
2022-05-19T13:40:17.841Z        debug   awsxrayexporter@v0.45.1/awsxray.go:99   Error translating span. {"kind": "exporter", "name": "awsxray", "error": "invalid xray traceid: ba4a6d6db3b5f7db2d1a910a988498bd"}
```

I am a bit surprised because I thought by using open telemetry I would work with a standardized set of APIs and tools.
Is this something that can be fixed in Quarkus OpenTelemetry, e.g. by a aws specific exporter implementation or only in the awsxrayexporter maintained by AWS?

**Comment by syr** [comment]
Just found that it is documented in the guide https://quarkus.io/guides/opentelemetry#id-generator 
However, as I was only working with the quickstart code I had no idea. 
@gsmet, @kenfinnigan @Sgitario  Anyone of you has a branch which is closer to the guide and could push it here, please?

**Comment by kenfinnigan** [comment]
Best to check with @radcortez

**Comment by radcortez** [comment]
Yes, that is the correct extension for the `AwsXrayIdGenerator`.

Not totally sure if I follow what you want to do. Do you want a quickstart with an example of using OpenTelemetry with AWS?
